src/components/App/App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import InputItem from "../InputItem";
+import * as list from "../ItemList"
```

Comment:
Лучше так не делать, а экспортировать переменные по именам. Что-то типа такого
```suggestion
import {ItemList, len as itemsCount} from "../ItemList"
```